muikku-timed-notifications-plugin/src/main/java/fi/otavanopisto/muikku/plugins/timed/notifications/strategies/AbstractTimedNotificationStrategy.java

```diff
@@ -13,12 +15,19 @@
 import javax.ejb.TimerService;
 import javax.inject.Inject;
 
+import org.apache.commons.lang3.StringUtils;
+
 import de.neuland.jade4j.JadeConfiguration;
 import de.neuland.jade4j.exceptions.JadeException;
 import fi.otavanopisto.muikku.jade.JadeController;
+import fi.otavanopisto.muikku.model.users.OrganizationEntity;
 import fi.otavanopisto.muikku.plugins.timed.notifications.TimedNotificationsJadeTemplateLoader;
+import fi.otavanopisto.muikku.schooldata.SchoolDataIdentifier;
+import fi.otavanopisto.muikku.users.OrganizationEntityController;
 
-public abstract class AbstractTimedNotificationStrategy implements TimedNotificationStrategy{
+public abstract class AbstractTimedNotificationStrategy implements TimedNotificationStrategy {
+  
+  private static final String ENABLED_ORGANIZATIONS = System.getProperty("muikku.timednotifications.enabledorganizations");
```

Comment:
Requires salt changes? Double-check that notifications continue to work normally even if this hasn't been set (which should be the case, given that it is just a missing search filter?)

Reply:
Done